game/domain/maneuver/draw_new_hand.lua

```diff
@@ -1,20 +1,23 @@
 
 local DEFS = require 'domain.definitions'
 
-local maneuver = {}
+local DRAWHAND = {}
 
-maneuver.schema = {}
+DRAWHAND.param_specs = {}
 
-function maneuver.validate(actor, sector, params)
+function DRAWHAND.activatedAbility(actor, sector, params)
+  return nil
+end
+
+function DRAWHAND.validate(actor, sector, params)
   return actor:isHandEmpty() and actor:getPP() >= DEFS.NEW_HAND_COST
 end
 
-function maneuver.perform(actor, sector, params)
-  actor:spendPP(DEFS.NEW_HAND_COST)
+function DRAWHAND.perform(actor, sector, params)
   for i = 1, DEFS.HAND_LIMIT do
```

Comment:
here, why did you remove, the `spendPP` line?

Reply:
Just an accident I'd guess.